### PR TITLE
[Feature][Doc] Add RocketMq connector

### PR DESCRIPTION
**Pull request by zhilinli123**
subTask:https://github.com/apache/seatunnel/issues/4572

### CODE REVIEW

File: docs/en/connector-v2/sink/RocketMQ.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # RocketMQ
 
 > RocketMQ sink connector
->
-  ## Description
+
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/RocketMQ.md:4]: Please add supported engine follow https://github.com/apache/seatunnel/pull/5132/files#diff-1adcb2672f956c38a665af1cb0b5dab052f2e1e8b822dc33032320ce1a3bceb2R5-R9

File: docs/en/connector-v2/source/RocketMQ.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 > RocketMQ source connector
 
+## Support Apache RocketMQ Version
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/RocketMQ.md:5]: Same as above.

File: docs/en/connector-v2/source/RocketMQ.md [hunk 3]
```diff
@@ -17,125 +21,84 @@ Source connector for Apache RocketMQ.
...
-### common-options [config]
+## Task Example
 
-Source plugin common parameters, please refer to [Source Common Options](common-options.md) for details.
+### Simple:
 
-## Example
-
-### Simple
+> random generation data is sent to Rocketmq
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/RocketMQ.md]: The describe not right.

Comment (zhilinli123) [reply docs/en/connector-v2/source/RocketMQ.md]: done,thanks review

File: docs/en/connector-v2/sink/RocketMQ.md [hunk 4]
```diff
@@ -1,44 +1,43 @@
...
+| acl.enabled          | Boolean | no       | false                    | false                                                                                                                                                               |
+| access.key           | String  | no       |                          | When ACL_ENABLED is true, access key cannot be empty                                                                                                                |
+| secret.key           | String  | no       |                          | When ACL_ENABLED is true, secret key cannot be empty                                                                                                                |
+| producer.group       | String  | no       | SeaTunnel-producer-Group | SeaTunnel-producer-Group                                                                                                                                            |
+| semantic             | string  | no       | NON                      | Semantics that can be chosen EXACTLY_ONCE/AT_LEAST_ONCE/NON, default NON.                                                                                           |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/RocketMQ.md]: Do you read the `RocketMqSinkFactory` code? Some options I don't found in the code.

image

**Review by EricJoy2048 — DISMISSED** [review]

File: docs/en/connector-v2/sink/RocketMQ.md (hunk 4 above)
Comment (zhilinli123) [reply docs/en/connector-v2/sink/RocketMQ.md]: Is this parameter `semantic`?
I found the rest in here
`org.apache.seatunnel.connectors.seatunnel.rocketmq.config.ProducerConfig`
`org.apache.seatunnel.connectors.seatunnel.rocketmq.config.Config`

**Comment by EricJoy2048** [comment]
It seems that many of the key options do not add examples to tell users how to use them. Can I add more examples?

**Comment by zhilinli123** [comment]
PTAL: @EricJoy2048 @Hisoka-X I made some changes

**Review by EricJoy2048 — APPROVED** [review]

**Review by liugddx — APPROVED** [review]
LGTM

**Pull request merged**